pkg/client/hsclient.go

```diff
@@ -450,7 +450,7 @@ func (client *HammerspaceClient) CreateShare(name string,
 	if len(exportOptions) == 0 { // add default export options to share create request
 			exportOptions = append(exportOptions, common.ShareExportOptions{
 			Subnet:            "*",
-			RootSquash:        true,
+			RootSquash:        false,
```

Comment:
I'm not really comfortable with leaving "*" unsquashed by default. Is there any reason why this is needed?